scripts/storybook/src/utils.js

```diff
@@ -222,9 +222,7 @@ function getImportMappingsForExportToSandboxAddon(allPackageInfo = getAllPackage
 function getPackageStoriesGlob(options) {
   const projects = getAllProjects();
 
-  const excludeStoriesInsertionFromPackages = (options.excludeStoriesInsertionFromPackages ?? []).map(packageName =>
```

Comment:
this is correct and won't solve the bug within the API as expected (exclude api should work with both project name and package name).

What needs to be done:

1. update test https://github.com/microsoft/fluentui/blob/master/scripts/storybook/src/utils.spec.js#L248

```diff
- expect(actual).not.toContain(expect.stringContaining('/react-text/stories/'));
+ expect(actual).not.toContainEqual(expect.stringContaining('/react-text/stories/'));
```

now it will fail


2. update logic

https://github.com/microsoft/fluentui/blob/master/scripts/storybook/src/utils.js#L242-L246

```diff
+ const projectName = normalizeProjectName(pkgName);
- if (!(pkgName.startsWith('@fluentui/') && !excludeStoriesInsertionFromPackages.includes(pkgName))) {
+ if (!(pkgName.startsWith('@fluentui/') || excludeStoriesInsertionFromPackages.includes(projectName))) {
-      return acc;
- }

- const projectName = normalizeProjectName(pkgName);
```

test passes and rendering works as expected

Reply:
Thanks for the feedback! Here are a few more suggestions:

1. We should ensure that our tests cover both package and project name cases.
2. There might be an error in the condition; it won't work as expected:
```
if (!(pkgName.startsWith('@fluentui/') || !excludeStoriesInsertionFromPackages.includes(pkgName))) {
but this should
```
 if (!pkgName.startsWith('@fluentui/') || excludeStoriesInsertionFromPackages.includes(projectName)) {
      return acc;
 }
```

Reply:
Updated PR with those changes

Reply:
> but this should

that's what I wrote no ? :D

<img width="796" alt="image" src="https://github.com/user-attachments/assets/a9b8ab01-6bf8-4603-8e88-656454637a5a">